src/pages/HomePage.jsx

```diff
@@ -11,6 +11,7 @@ import { globalStore } from "../stores";
  * - 로그인하지 않은 사용자가 게시물에 좋아요를 누를 경우, "로그인 후 이용해주세요"를 alert로 띄운다.
  */
 export const HomePage = () => {
+  const { loggedIn } = globalStore.getState();
   const { posts } = globalStore.getState();
```

Comment:
같은 곳에서 state를 가져오는 것 같은데 아래처럼 하는게 더 나을 것 같아요ㅎㅎ

```suggestion
  const { posts, loggedIn } = globalStore.getState();
```